project/Ydoc.scala

```diff
@@ -86,12 +89,23 @@ object Ydoc {
         if (changed || !nodeModules.isDirectory) {
           val command = s"$pnpmCommand i --frozen-lockfile"
           streams.log.info(command)
-          command ! streams.log
+          val exitCode = command ! streams.log
+          if (exitCode != 0) {
+            throw new CommandFailed(command, exitCode)
+          }
         }
     }
 
-    val inputFile = base / "package-lock.json"
+    val inputFile = base / "pnpm-lock.json"
```

Comment:
Ydoc build was failing on CI because inputs were wrong. We don't have `package-lock.json` anymore.